app/_src/introduction/kuma-requirements.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Kuma Requirements
+---
+
+This page expose the different requirements to run {{site.mesh_product_name}}.
+
+## Architecture
+
+{{site.mesh_product_name}} supports machines with `x86_64` architecture and since `1.7.0` it's possible to run {{site.mesh_product_name}} on `arm64`.
+
+## Kubernetes
+
+{{site.mesh_product_name}} is validated against Kubernetes `1.19.x`, `1.20.x`, `1.21.x` and `1.22.x`.
```

Comment:
We test on `1.26.1` and `1.20.15` - see [circleci config](https://github.com/kumahq/kuma/blob/master/.circleci/config.yml#L7-L8). Also this would get out of date every time we bump the version there. So would be good to have some sort of synchronising this or maybe just linking to the config?

Reply:
I added the correct version, and linked the config. Automation I think is out of scope here.